data/Arts2023Longitudinal.yaml

```diff
@@ -0,0 +1,3855 @@
+# yaml-language-server: $schema=.schema.yaml
+title: Longitudinal and quantitative fecal shedding dynamics of SARS-CoV-2, pepper
+  mild mottle virus, and crAssphage
+doi: 10.1128/msphere.00132-23
+description: >
+  The authors present longitudinal, quantitative fecal shedding data for SARS-CoV-2
+  RNA, pepper mild mottle virus (PMMoV) RNA, and crAss-like phage (crAssphage) DNA
+  from 48 COVID-19 patients. Abundances were quantified using (RT)-ddPCR assays targeting
+  the N and ORF1a genes. The data were contributed by the corresponding author.
```

Comment:
Looks like the corresponding author is Krista Wigginton for this paper. We should update this line to report where we got the data from (e.g., obtained from supplementary material, extracted from figure, etc.).

Reply:
Thank you for reviewing my pull request and catching all those mistakes!

Reply:
Pleasure! Tests can catch a lot of things, but having humans look over data is still one of the best ways of getting things into shape.

Reply:
Detailed reviews like yours really help me improve. Thanks for your thorough feedback!